### PR TITLE
Modernize supported browser documentation.

### PR DESCRIPTION
**Pull request by apetro**
Riffing on [uportal-dev@ discussion](https://groups.google.com/a/apereo.org/d/topic/uportal-dev/ipZKN9THoBM/discussion) of what browsers uPortal 5 ought to support.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,6 +60,10 @@ Consider pushing your changes to a topic branch in your (public) fork of the rep
 
 _Prior to committing, if you want to pull in the latest upstream changes  (highly appreciated btw), please use rebasing rather than merging.  Merging creates "merge commits" that really muck up the project timeline._
 
+## Test
+
+If your change is to a user-facing experience, it should not regress support for [supported browser versions](http://jasig.github.io/uPortal/SUPPORTED_BROWSERS.html).
```
Comment (ChristianMurphy) [inline CONTRIBUTING.md]: :art: Relative links usually work better, it allows the parser to determine how the link should be expanded.

``` md
[supported browser versions](SUPPORTED_BROWSERS.md)
```

**Comment by bjagg** [comment]
+1

File: docs/SUPPORTED_BROWSERS.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Policy
```
Comment (ChristianMurphy) [inline docs/SUPPORTED_BROWSERS.md]: :art: Usually its best to have a single h1 header

``` md
# Supported Browsers

## Policy

...

## Practice

...

## Rational

...
```

File: docs/SUPPORTED_BROWSERS.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Supported Browsers
+
+## Policy
+
+uPortal supports **the latest general audience release** and **one prior release** of these major browsers:
+
++ Chrome (on Android, MacOS, Windows) (including support for TalkBack)
++ Firefox (on MacOS and on Windows)
```
Comment (cousquer) [inline docs/SUPPORTED_BROWSERS.md]: Could you add Linux to Firefox ? something like:
`Firefox (on MacOS, IOS, Android, Windows, Linux) (including support for ChromeVox)`
Sometimes Linux users get upset very very quickly if they feel they are being forgotten.

File: docs/SUPPORTED_BROWSERS.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Supported Browsers
+
+## Policy
+
+uPortal supports **the latest general audience release** and **one prior release** of these major browsers:
+
++ Chrome (on Android, MacOS, Windows) (including support for TalkBack)
```
Comment (cousquer) [inline docs/SUPPORTED_BROWSERS.md]: Chrome (on Android, MacOS, IOS, Windows) (including support for TalkBack)

File: docs/SUPPORTED_BROWSERS.md (hunk 3 above)
Comment (apetro) [inline docs/SUPPORTED_BROWSERS.md]: Here's the thing: No browser support fairy comes along and makes it true that uPortal actually supports the browsers targeted in this policy.  Supporting browsers is work. I think the best strategy for uPortal is to commit to as little as we can get away with and then really fulfill those commitments. 

If it proves feasible, sustainable to fulfill a more modest browser support policy, then sure, uPortal could look to make bigger commitments. :)

**Pull request merged**